plugins/techdocs/src/reader/components/TechDocsPageHeader.tsx

```diff
@@ -59,6 +60,8 @@ export const TechDocsPageHeader = ({
     ownerEntity = parseEntityName(owner, { defaultKind: 'group' });
   }
 
+  const docsRootLink = useRouteRef(rootRouteRef)();
```

Comment:
is this an extra pair of parens?

Reply:
Yes - `useRouteRef` returns a `RouteFunc`

![image](https://user-images.githubusercontent.com/8904624/110951070-e9e44c80-8344-11eb-9f99-30f3e58edb0f.png)

Reply:
ah I see; learning new things every day, thank you @tudi2d !